README.md

```diff
@@ -1,40 +1,274 @@
 
 # UmikoBot
-## Prerequisites
-### Windows
-- [Qt](https://www.qt.io/) (`> 5`)
-- [OpenSSL](https://indy.fulgan.com/SSL/Archive/Experimental/openssl-1.0.2o-x64-VC2017.zip) (`OpenSSL v1.0.2`)
- - Visual Studio 2017 or 2019
- - Some sort of bash
 
-### Linux
-To build on Linux you're going to need to have Qt installed which is done by installing `qt5-default` and `libqt5websockets5-dev`.
+Umiko is the bot used on [TheCherno's Discord Sever](https://discord.gg/K9rMPsu9pQ). This repository is what makes Umiko; it's the source code and it is completely open source!
 
-## Setting up the project
+## ❓ What can it do?
 
-To build Umiko you have to first run the init script (`./init.sh`).
+Umiko can do everything that a Discord bot can technically do. Of course it doesn't do all of it, but if you feel like there's something missing that would make Umiko better, contributions are always welcomed!
 
-- On Windows you would be asked the paths for **x86** and **x64** versions of Qt, and OpenSSL. You don't need both **x86** and **x64** - you can just use one and build for that specific platform.
+## ℹ️️ What does it do?
 
-- After that you can get the project files by running `generate_project_files.sh` with the required target.
+Umiko is mainly built for fun. Currently, Umiko can
 
-  - On Windows you can just run the script normally; It will create the project for VS 2017.
+- Keep track of your Currency and XP on the server: Both the Currency and XP system are extensively developed and are backed by a multitude of commands.
+- Show you memes from the internet.
+- Show you random GitHub repositories because why not?
+- Keep track of your timezone.
+- Keep track of your information so that others can use a single command to know (*everything* :eyes:) about you.
+- Use your XP to assign you levels so that there's a bit of a competitive spark.
+- Reward you for typing `<prefix>daily`, well, daily.
+- Crash and work unexpectedly due to zillions of bugs here and there...
+- Reward you with some (very useful) currency because you typed and you were lucky enough.
+- Host events which tweak with Umiko's normal behavior to keep the fun going.
+  
+...and many more features still in works!
 
-  - On Linux you are going to need to build it with **qmake** (to get it just install `qt5-qmake`).
+## 🙋‍♂️ Contributing
 
-## Building
+So you want to contribute? Awesome! Let's get you going. For starters, know that Umiko uses the **[QDiscord API](https://github.com/Gaztin/QDiscord)** to interface with Discord. This is already in the repo as a submodule, so you don't need to worry much.
```

Comment:
QDiscord is not an API. It's a library that interfaces with the *Discord API*!

Reply:
Woops, well yeah I meant that. Fixing it rn!